lib/style.dart

```diff
@@ -175,6 +175,20 @@ class Style {
   Alignment? alignment;
   String? markerContent;
 
+  /// MaxLine
+  ///
+  ///
+  ///
+  ///
+  int? maxLine;
```

Comment:
I suggest renaming this to `maxLines` to be more inline with other Flutter widgets